options/options.js

```diff
@@ -49,7 +49,7 @@ $(() => {
     $('#login-user').text(`${user}${domain}`).attr('href', userLink);
     $('#token').attr('href', tokenLink);
   })
-  .catch((err) => {
+  .catch(err => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/3bad4bc025cc43cc251f17b68f22b94ac54ac1f0/options/options.js#L52"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>